planemo/workflow_lint.py

```diff
@@ -59,6 +60,12 @@ class WorkflowLintContext(LintContext):
     training_topic = None
 
 
+def build_wf_lint_args(ctx, **kwds):
+    lint_args = build_lint_args(ctx, **kwds)
+    lint_args["iwc_grade"] = str(kwds.get("iwc", False))
```

Comment:
```suggestion
    lint_args["iwc_grade"] = kwds.get("iwc", False)
```

We should just leave this a boolean.

Reply:
If I keep it as boolean then I get a type violation as lint_args is supposed to be a dict of str -> str

Reply:
See https://github.com/galaxyproject/planemo/blob/b9e62ef323766841b2d951c367493687d833c77d/planemo/workflow_lint.py#L125

Reply:
`Dict[str, Any]` is probably enough, I'll push a commit to your branch.

Reply:
Thanks, if not I can commit it.